src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js

```diff
@@ -3,30 +3,30 @@ import styled from 'styled-components';
 export const Container = styled.div`
   display: flex;
   flex-flow: wrap;
-  div:nth-child(odd) {
+  & > div:nth-child(odd) {
     background: #f6f6f6;
   }
-  div:nth-child(n+4) {
+  & > div:nth-child(n + 4) {
     margin-top: 10px;
   }
-`
+`;
 
 export const TrafficSourceContainer = styled.div`
   flex: calc(100% / 3);
-  box-shadow: 2px 0 4px 0 rgba(0,0,0,0.2);
-  border-bottom: 1px solid #CCC;
-  border-top: 1px solid #CCC;
-`
+  box-shadow: 2px 0 4px 0 rgba(0, 0, 0, 0.2);
+  border-bottom: 1px solid #ccc;
```

Comment:
I see this is very nice, but maybe some stuff like repeated colors can be stored in a variable for this segment of css. Since this is a template string literal, perhaps you could add a var like this `const gray = '#CCC'` and then use this in the border property like this `border-bottom: 1px solid ${gray}`. What do you think? Perhaps I'm overthinking...

Reply:
we talked in slack. For now it's not necessary we will wait for later if many styles are shared